src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.cs

```diff
@@ -526,10 +562,12 @@ private async Task EnqueueProjectConfigurationChangeWorkItemAsync(ProjectChanges
                 // TODO: why solution changes return Project not ProjectId but ProjectChanges return DocumentId not Document?
                 var projectConfigurationChange = UnitTestingInvocationReasons.Empty;
 
+#if false // Not used in unit testing crawling
                 if (!object.Equals(oldProject.ParseOptions, newProject.ParseOptions))
                 {
                     projectConfigurationChange = projectConfigurationChange.With(UnitTestingInvocationReasons.ProjectParseOptionChanged);
```

Comment:
This seems a bit dicey. I think test discovery should probably treat a change in project parse options as a change in project configuration (and retrigger analysis for the project). So, it may be nice to preserve this.

However, I guess your response would be that we are welcome to uncomment this if needed :)

Reply:
Please ignore the above comment. Your explanation for the other related comment above clarifies how this is supposed to work.